articles/application-insights/app-insights-dashboards.md

```diff
@@ -95,6 +95,9 @@ When you've created a dashboard, you can share it with other users.
 
 Learn about [Roles and access control](app-insights-resources-roles-access-control.md).
 
+## Create dashboards programmatically
+You can automate dashboards creation using [Azure Resource Manager](https://docs.microsoft.com/azure/azure-portal/azure-portal-dashboards-create-programmatically) and a simple JSON editor.
```

Comment:
One small fix. Remove the **s** from dashboards in line 99. Change "You can automate dashboards creation using" to "You can automate dashboard creation using" and then this will be all set. The plural form works in the H2, but it needs to be singular in the context of your next sentence.